res/values-no/strings.xml

```diff
@@ -882,7 +882,7 @@
     <string name="Free_Agent">Fri Agent</string>
     <string name="This_player_selected_a_different_teammate_">Denne spilleren valgte en annen lagkamerat.</string>
     <string name="muted_until">Muted until</string>
-    <string name="Content_rating_of_the_app_may_change_when_playing_multiplayer">Content rating of the app may change when playing multiplayer.</string>
+    <string name="Content_rating_of_the_app_may_change_when_playing_multiplayer">innholdsvurderingen av appen kan endres når du spiller flerspiller.</string>
```

Comment:
The first vowel "I" in "innholdsvurderingen" should be capital letter.

Reply:
done